typings/index.d.ts

```diff
@@ -97,6 +97,9 @@ declare module 'ytsr' {
       }[];
       currentRef?: string;
     }
+
+    function getFilters(searchString: string, callback?: Function): Promise<any>
```

Comment:
Maybe this would be better, I'm not sure. Feel free to commit suggestion if it is.
```suggestion
    function getFilters(searchString: string, callback?: Function): Promise<Map<string, any>>
```